README.md

```diff
@@ -1,4 +1,5 @@
-<img src="https://qwertyquerty.github.io/pypresence/html/_static/logosmall.svg" width="100%">
+
+[comment]: <> (<img src="https://qwertyquerty.github.io/pypresence/html/_static/logosmall.svg" width="100%">)
```

Comment:
Unnecessary, fine as is..

Reply:
![image](https://user-images.githubusercontent.com/21298211/90547216-24905880-e159-11ea-8502-a7379325fa95.png)
Not really fine as is

Reply:
Qwerty's image fucked off apparently, that's a him problem @qwertyquerty

Reply:
That's why I commented it...